src/Service.Tests/Unittests/MultipleCreateUnitTests/MultipleCreateOrderHelperUnitTests.cs

```diff
@@ -374,6 +375,23 @@ public void ValidateReferencingEntityBasedOnEntityMetadata()
 
         #endregion
 
+        #region Order determination test for relationships having source/target entities backed by same database table
+
+        /// <summary>
+        /// Test to validate that when multiple-create is executed for a relationship for which source and target entities are backed by the
+        /// same database tables, we throw an appropriate exception because DAB currently does not support multiple-create for such relationships.
```

Comment:
```suggestion
        /// same database table, we throw an appropriate exception because DAB currently does not support multiple-create for such relationships.
```